apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -190,7 +191,7 @@ export class AppRoomBridge extends RoomBridge {
 	private async getUsersByRoomIdAndSubscriptionRole(roomId: string, role: string): Promise<IUser[]> {
 		const subs = await Subscriptions.findByRoomIdAndRoles(roomId, [role], { projection: { uid: '$u._id', _id: 0 } });
 		// Was this a bug?
-		const users = await Users.findByIds(subs.map((user) => user.u._id));
+		const users = await Users.findByIds(subs.map((user: { uid: string }) => user.uid));
```

Comment:
@KevLehman I intentionally reverted your change here, I believe it was indeed correct if you look at the projection at the above line. @RocketChat/apps can you please confirm that?